spec/controllers/admin/roles_controller_spec.rb

```diff
@@ -20,6 +29,14 @@
       expect(JSON.parse(response.body)['data'].pluck('id')).to match_array(roles.pluck(:id))
     end
 
+    it 'admin without ManageRoles permission cannot return the list of roles' do
```

Comment:
Since GL3 is permission based, there's no meaning behind `admin` role,
I'd change the message to `user without "ManageRoles" ...` instead to avoid giving a meaning to the admin role,